src/timesync/fuzz-nts-extfields.c

```diff
@@ -0,0 +1,80 @@
+#include <string.h>
+#include <stdio.h>
+#include <assert.h>
+#include <fcntl.h>
+#include <unistd.h>
+
+#include "fuzz.h"
+
+#include "nts.h"
+#include "nts_crypto.h"
+#include "nts_extfields.h"
+
+static void eat(volatile const uint8_t* buf, size_t size) {
+        if (buf) while (size) (void)buf[size--];
+}
+
+/* this program does no sanity checking as it is meant for fuzzing only */
+int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
+        uint8_t buffer[1280];
+        int len = size > sizeof(buffer)? sizeof(buffer) : size;
+
+        memcpy(buffer, data, len);
+        if (len < 48) return 0;
+
+        struct NTS_Query nts = (struct NTS_Query) {
+                .cipher = *NTS_get_param(NTS_AEAD_AES_SIV_CMAC_256),
+                .c2s_key = (void*)"01234567890abcdef",
+                .s2c_key = (void*)"01234567890abcdef",
+        };
+
+        /* fuzz the NTS extension field parser */
+        struct NTS_Receipt rcpt = { 0, };
```

Comment:
please just use `= {};`, i.e. empty brackets. That is only allowed as a gnu extension, but it is one we use heavily.